src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -898,18 +899,25 @@ public void uploadArcResultWithScheduleContext() throws Exception {
         StudyParticipant participant = participantsApi.getParticipantById(userId, false).execute().body();
         String healthCode = participant.getHealthCode();
 
-        // Give participant  data group.
+        // This test looks at study2 instead of study1, so we need a bit of extra time before the participant version
+        // is available in study2. Since the participant version doesn't get created until we change the sharing
+        // status from no_sharing to all_qualified_researchers, so we'll do this now and insert a wait to give us time
```

Comment:
```suggestion
        // status from no_sharing to all_qualified_researchers, we'll do this now and insert a wait to give us time
```
(or drop the "Since" at the beginning of the sentence, on the previous line)